src/Kafka/CreateKafkaInstance/components/InstanceInfo.tsx

```diff
@@ -53,126 +59,163 @@ export const InstanceInfo: VoidFunctionComponent<
-                  value: storage,
-                })}
-              </TextListItem>
-            </GridItem>
-            <GridItem>
-              <TextListItem component={TextListItemVariants.dt}>
-                {t("kafka:partitions")}
-              </TextListItem>
-              <TextListItem component={TextListItemVariants.dd}>
-                {t("create-kafka-instance:partitions_value", {
-                  value: maxPartitions,
-                })}
-              </TextListItem>
-            </GridItem>
-            <GridItem>
-              <TextListItem component={TextListItemVariants.dt}>
-                {t("kafka:client_connections")}
-              </TextListItem>
-              <TextListItem component={TextListItemVariants.dd}>
-                {t("create-kafka-instance:client_connections_value", {
-                  value: connections,
-                })}
-              </TextListItem>
-            </GridItem>
-            <GridItem>
-              <TextListItem component={TextListItemVariants.dt}>
-                {t("kafka:connection_rate")}
-              </TextListItem>
-              <TextListItem component={TextListItemVariants.dd}>
-                {t("create-kafka-instance:connection_rate_value", {
-                  value: connectionRate,
-                })}
-              </TextListItem>
-            </GridItem>
-            <GridItem>
-              <TextListItem component={TextListItemVariants.dt}>
-                {t("kafka:message_size")}
-              </TextListItem>
-              <TextListItem component={TextListItemVariants.dd}>
-                {t("create-kafka-instance:message_size_value", {
-                  value: messageSize,
-                })}
-              </TextListItem>
-            </GridItem>
-          </Grid>
-        )}
-        {isLoading && (
-          <Grid sm={6} lg={12} hasGutter>
-            <GridItem>
-              <Skeleton width="75%" screenreaderText="Loading contents" />
-            </GridItem>
-            <GridItem>
-              <Skeleton width="66%" />
-            </GridItem>
-            <GridItem>
-              <Skeleton width="50%" />
-            </GridItem>
-            <GridItem>
-              <Skeleton width="33%" />
-            </GridItem>
-            <GridItem>
-              <Skeleton width="25%" />
-            </GridItem>
-            <GridItem>
-              <Skeleton width="55%" />
-            </GridItem>
-            <GridItem>
-              <Skeleton width="35%" />
-            </GridItem>
-          </Grid>
-        )}
-        <Button
-          isSmall
-          isInline
-          variant={ButtonVariant.link}
-          style={{ marginTop: "20px" }}
-          onClick={onClickQuickStart}
-        >
-          {t("create-kafka-instance:quick_start_guide_message")}
-        </Button>
-      </TextList>
+          </TextList>
+        </CardBody>
+      </Card>
+      <Card isCompact className="pf-u-mt-md">
```

Comment:
Instead of using utility classes like this, we can place the `<Card>`s into a Stack layout with a gutter.